tests/test_config/test_get_final_config_sphinx_recurse_.yml

```diff
@@ -0,0 +1,94 @@
+_user_config:
+  sphinx:
+    config:
+      html_theme: other
+      html_theme_options:
+        launch_buttons:
+          binderhub_url: other
+      new: value
+    extra_extensions:
+    - other
+    local_extensions:
+      helloworld: ./ext
+    recursive_update: true
+final:
+  author: The Jupyter Book community
+  comments_config:
+    hypothesis: false
+    utterances: false
+  copyright: '2021'
+  exclude_patterns:
+  - '**.ipynb_checkpoints'
+  - .DS_Store
+  - Thumbs.db
+  - _build
+  execution_allow_errors: false
+  execution_excludepatterns: []
+  execution_in_temp: false
+  execution_timeout: 30
+  extensions:
+  - sphinx_togglebutton
+  - sphinx_copybutton
+  - myst_nb
+  - jupyter_book
+  - sphinx_thebe
+  - sphinx_comments
+  - sphinx_external_toc
+  - sphinx.ext.intersphinx
+  - sphinx_panels
+  - sphinx_book_theme
+  - other
+  - helloworld
+  - sphinx_jupyterbook_latex
+  external_toc_exclude_missing: false
+  html_baseurl: ''
+  html_favicon: ''
+  html_logo: ''
+  html_sourcelink_suffix: ''
+  html_theme: other
+  html_theme_options:
+    extra_footer: ''
+    extra_navbar: Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
+    google_analytics_id: ''
+    home_page_in_toc: true
+    launch_buttons:
+      binderhub_url: other
```

Comment:
just to confirm - the main thing we're checking is that all of the other default `html_theme_options` are here, because we're recursively updated the config. Is that right?

Reply:
Yes.  That's the idea, at least.